src/components/block-gallery/gallery-image.js

```diff
@@ -175,14 +181,37 @@ class GalleryImage extends Component {
 		return (
 			<figure className={ 'coblocks-gallery--figure ' + className } tabIndex="-1" onKeyDown={ this.onKeyDown } ref={ this.bindContainer }>
 				{ isSelected &&
-					<div className="components-coblocks-remove-gallery-item-button-wrapper">
-						<IconButton
-							icon="no-alt"
-							onClick={ onRemove }
-							className="components-coblocks-remove-gallery-item-button-button"
-							label={ __( 'Remove Image' ) }
-						/>
-					</div>
+					<Fragment>
+						{ supportsMoving &&
+							<div className="components-coblocks-gallery-item__move-menu">
+								<IconButton
+									icon={ verticalMoving ? "arrow-up" : "arrow-left" }
+									onClick={ isFirstItem ? undefined : onMoveBackward }
```

Comment:
Instead of using an if-else ternary here we can utilize the `&&` operator and remove the `undefined` return value.

```suggestion
									onClick={ isFirstItem && onMoveBackward }
```

If `isFirstItem` is true and `onMoveBackward` is defined, `onMoveBackward` will be returned here. If `isFirstItem` is false then false will be returned.

Reply:
I used `onClick={ ! isFirstItem && onMoveBackward }`

Reply:
Good catch! :)